packages/driver/cypress/e2e/commands/traversals.cy.js

```diff
@@ -346,17 +346,28 @@ describe('src/cy/commands/traversals', () => {
       const button = cy.$$('#button').hide()
 
       cy.on('fail', (err) => {
-        const log = this.logs[1]
+        assertLogLength(this.logs, 3)
 
-        expect(log.get('state')).to.eq('failed')
-        expect(err.message).to.include(log.get('error').message)
-        expect(log.get('$el').get(0)).to.eq(button.get(0))
+        const getLog = this.logs[0]
+        const findLog = this.logs[1]
+        const assertionLog = this.logs[2]
 
-        const consoleProps = log.invoke('consoleProps')
+        expect(err.message).to.contain('This element `<button#button>` is not visible because it has CSS property: `display: none`')
+
+        expect(getLog.get('state')).to.eq('passed')
+        expect(getLog.get('error')).to.be.undefined
+
+        expect(findLog.get('state')).to.eq('passed')
+        expect(findLog.get('error')).to.be.undefined
+        expect(findLog.get('$el').get(0)).to.eq(button.get(0))
+        const consoleProps = findLog.invoke('consoleProps')
 
         expect(consoleProps.Yielded).to.eq(button.get(0))
         expect(consoleProps.Elements).to.eq(button.length)
 
+        expect(assertionLog.get('state')).to.eq('failed')
+        expect(err.message).to.include(assertionLog.get('error').message)
```

Comment:
Found with this test, we had missing test cases in query.cy.ts, hence the added tests

| Before | After |
| - | - |
| <img width="605" alt="Screen Shot 2022-11-03 at 8 42 42 AM" src="https://user-images.githubusercontent.com/14099737/199736692-eed0cdca-412a-45f3-b578-e0d83214f1c7.png">|<img width="614" alt="Screen Shot 2022-11-03 at 8 43 09 AM" src="https://user-images.githubusercontent.com/14099737/199736693-04e635cd-2652-4a57-83cf-c554bb17fca1.png">